xgboost-predictor/src/main/java/biz/k11i/xgboost/tree/DefaultRegTreeFactory.java

```diff
@@ -9,7 +9,7 @@ public final class DefaultRegTreeFactory implements RegTreeFactory {
   public static RegTreeFactory INSTANCE = new DefaultRegTreeFactory();
 
   @Override
-  public final RegTree loadTree(ModelReader reader) throws IOException {
+  public RegTree loadTree(int treeId, ModelReader reader) throws IOException {
```

Comment:
`treeId` seems to me not used. 

Am I missing something? Forgive me if I look into it a bit early, I know its a draft

Reply:
it is used in the companion PR in h2o-3 repo - it is needed to identify the trees to skip in iterative scoring with caching